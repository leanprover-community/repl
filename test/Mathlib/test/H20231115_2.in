{"cmd": "import Mathlib.Tactic.Cases"}

{"cmd": "example {x : Nat} : x + 1 > x := by\n  induction' x with x hx\n  all_goals sorry", "env": 0, "sorries": "individual"}

