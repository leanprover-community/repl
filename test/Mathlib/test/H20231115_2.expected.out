{"env": 0}

{"sorries":
 [{"proofState": 0,
   "pos": {"line": 3, "column": 12},
   "parentDecl": "_example",
   "goals": ["case zero\n⊢ 0 + 1 > 0"],
   "endPos": {"line": 3, "column": 17}},
  {"proofState": 1,
   "pos": {"line": 3, "column": 12},
   "parentDecl": "_example",
   "goals": ["case succ\nx : Nat\nhx : x + 1 > x\n⊢ x + 1 + 1 > x + 1"],
   "endPos": {"line": 3, "column": 17}}],
 "messages":
 [{"severity": "warning",
   "pos": {"line": 1, "column": 0},
   "endPos": {"line": 1, "column": 7},
   "data": "declaration uses 'sorry'"}],
 "env": 1}

